mrc/localization/color/defaults.py

```diff
@@ -0,0 +1,13 @@
+from enum import Enum
+
+
+class DefaultColorValues(Enum):
+    """
+    Enum mapping color names into their hue (HSV color model) value from range [0, 360).
+    """
+    red = range(-30, 30)
+    yellow = range(30, 90)  # (100, 100, 0)
```

Comment:
How about adding information about encoding of commented colors for better readibility?